vocabularies/Common.xml

```diff
@@ -1118,6 +1118,11 @@ only `TargetProperties` and `TargetEntities` are relevant. They are addressed vi
           <Annotation Term="Core.Description" String="Indicates whether the side-effect has already happened" />
           <Annotation Term="Core.LongDescription" String="The value of this property typically is a Path expression pointing to a boolean property. It can be used by clients to defer expensive refresh calls until they are actually needed and instead just request the referenced indicator property. Servers can choose to return indicator properties even if not explicitly requested." />
         </Property>
+        <Property Name="Discretionary" Type="Edm.Boolean" Nullable="true">
+          <Annotation Term="Common.Experimental" />
+          <Annotation Term="Core.Description" String="Indicates whether the client can decide if a side-effect should be triggered or not" />
```

Comment:
```suggestion
          <Annotation Term="Core.Description" String="If `true` then the client can decide if a side-effect should be triggered or not" />
```
Clarify what happens for `null` value.

Reply:
Or make the property `Nullable="false" DefaultValue="false"`.

Reply:
I took over the modelling from the TriggeredIndicator which somehow is similar and optional and usually "false". So I wonder if we also should change this one. Together with a default the annotation is still optional to be added and thus compatible, right?

Reply:
Correct. Better also change the `TriggeredIndicator`.

Reply:
`Nullable="false"` might fail if the value is computed via a `Path`.

Reply:
Ok, but for the new term I do not expect a path expression, but rather a static information, i.e. true / false.

Reply:
But you say otherwise:
https://github.com/SAP/odata-vocabularies/blob/5c73c570bb7283847a02009fa965c7e7181b6de0/vocabularies/Common.xml#L1124

Reply:
Yes, copy and paste is not good, just changed it.